bigchaindb/common/schema/transaction.yaml

```diff
@@ -205,19 +202,22 @@ definitions:
             type_id:
               type: integer
           description: |
-              Fulfillment of a condition_, or put a different way, this is a
-              payload that satisfies a condition in order to spend the associated
-              asset.
+              Fulfillment of an `Output.condition`_, or put a different way, this is a
+              payload that satisfies an output condition in order to spend
```

Comment:
We're using "spend" in quite a few places*, if it's wrong we should examine it separately imo.

* Example: https://github.com/bigchaindb/bdb-transaction-cli/blob/master/docs/examples.rst#spend